app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/NetworkInspector/NetworkTable/NoTrafficCTA.jsx

```diff
@@ -180,8 +180,7 @@ const NoTrafficCTA = ({ isStaticPreview }) => {
     navigate,
     numberOfConnectedApps,
     openConnectedAppsModal,
-    systemWideSource.isActive,
-    systemWideSource.isAvailable,
+    systemWideSource,
```

Comment:
Put `systemWideSource.isActive` and `systemWideSource.isAvailable,` in dependency array instead of whole object, if sure that we will get stable `systemWideSource` object for each rerender then its okay.